libs/zzz/formula-ui/src/components/TagDisplay.tsx

```diff
@@ -1,7 +1,7 @@
 import { ColorText } from '@genshin-optimizer/common/ui'
 import type { Tag } from '@genshin-optimizer/zzz/formula'
-import { getVariant, tagFieldMap } from '..'
-export function OptimizationTargetDisplay({ tag }: { tag: Tag }) {
+import { getVariant, tagFieldMap } from '../char'
+export function TagDisplay({ tag }: { tag: Tag }) {
   return (
     <ColorText color={getVariant(tag)}>
       {tagFieldMap.subset(tag)[0]?.title || tag.name || tag.q}
```

Comment:
should only use tag.name for the top-level formula, probably

Reply:
No way to know from this component.

Reply:
![thanos-fine-ill-do-it-myself](https://github.com/user-attachments/assets/b7b3ef6d-9a2e-4551-9d9c-d2c4020cff8f)